### PR TITLE
Update privacy.md

### PR DESCRIPTION
**Pull request by heiglandreas**
Make it more readable

### CODE REVIEW

File: web/privacy.html [hunk 1]
```diff
@@ -121,17 +120,17 @@ <h4>3. Cookies</h4>
     shopping cart in an online shop. The online store remembers the articles
     that a customer has placed in the virtual shopping cart via a cookie.</p>
 
-<p>The data subject may, at any time, prevent the setting of cookies through
+<p>TYou may, at any time, prevent the setting of cookies through
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
<p>You may, at any time, prevent the setting of cookies through
```

File: web/privacy.html [hunk 2]
```diff
@@ -11,26 +11,25 @@ <h4>Privacy Policy</h4>
...
+    of personal data; however, if you want to use special 
     services via our website, processing of personal data will become
     necessary. If the processing of personal data is necessary and there is
-    no statutory basis for such processing, we generally obtain consent from
-    the data subject.</p>
+    no statutory basis for such processing, we generally obtain consent from you.</p>
 
-<p>The processing of personal data, such as the name or e-mail address
-    of a data subject shall always be in line with the General Data Protection
+<p>The processing of personal data, such as yourname or e-mail address
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
<p>The processing of personal data, such as your name or e-mail address
```

File: web/privacy.html [hunk 3]
```diff
@@ -86,11 +86,10 @@ <h4>2. Name and Address of the controller</h4>
 
 </p>
 
-<address>joind.in
-<br/>Forsthausstraße 7
-<br/>61279 Grävenwiesbach
-<br/>Germany
-<br/>Phone: +49 6083 959030
+<address>The Joindin Foundationjoind.in
```
Comment (ericpoe) [inline web/privacy.html]: I believe that we're just "The Joindin Foundation"

File: web/privacy.html [hunk 4]
```diff
@@ -86,11 +86,10 @@ <h4>2. Name and Address of the controller</h4>
...
+<br/>PO Box 342457 
+<br/>Memphis, TN 38184
+<br/>Uniteds States of America
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
<br/>United States of America
```

File: web/privacy.html [hunk 5]
```diff
@@ -211,22 +199,22 @@ <h4>7. Routine erasure and blocking of personal data</h4>
...
+<h4>8. Your Rights</h4>
 <ul style="list-style: none;">
     <li><h4>a) Right of confirmation</h4>
-        <p>Each data subject shall have the right granted by the European
+        <p>You shall have the right granted by the European
             legislator to obtain from the controller the confirmation as to
             whether or not personal data concerning him or her are being
-            processed. If a data subject wishes to avail himself of this
-            right of confirmation, he or she may, at any time, contact any
+            processed. If you wishes to avail yourself of this
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
            processed. If you wish to avail yourself of this
```

File: web/privacy.html [hunk 6]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
+        <p>Furthermore, you shall have a right to obtain
             information as to whether personal data are transferred to a third
             country or to an international organisation. Where this is the
-            case, the data subject shall have the right to be informed of the
+            case, you shall have the right to be informed of the
             appropriate safeguards relating to the transfer.</p>
 
-        <p>If a data subject wishes to avail himself of this right of access,
-            he or she may, at any time, contact any employee of the controller.</p>
+        <p>If you wishes to avail yourself of your right of access,
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
        <p>If you wish to avail yourself of your right of access,
```

File: web/privacy.html [hunk 7]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
 
         <ul style="list-style: none;">
             <li>The personal data are no longer necessary in relation to the purposes for which they were collected or otherwise processed.</li>
-            <li>The data subject withdraws consent to which the processing is based according to point (a) of Article 6(1) of the GDPR, or point (a) of Article 9(2) of the GDPR, and where there is no other legal ground for the processing.</li>
-            <li>The data subject objects to the processing pursuant to Article 21(1) of the GDPR and there are no overriding legitimate grounds for the processing, or the data subject objects to the processing pursuant to Article 21(2) of the GDPR. </li>
+            <li>TYou withdraw consent to which the processing is based according to point (a) of Article 6(1) of the GDPR, or point (a) of Article 9(2) of the GDPR, and where there is no other legal ground for the processing.</li>
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
            <li>You withdraw consent to which the processing is based according to point (a) of Article 6(1) of the GDPR, or point (a) of Article 9(2) of the GDPR, and where there is no other legal ground for the processing.</li>
```

File: web/privacy.html [hunk 8]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
+        <p>you shall have the right granted by the European legislator to withdraw your consent to processing of his or her personal data at any time. </p>
 
-        <p>f the data subject wishes to exercise the right to withdraw the consent, he or she may, at any time, contact any employee of the joind.in.</p>
+        <p>If you wish to exercise the right to withdraw the consent, you may, at any time, contact the joind.in-Team via the contact form.</p>
 
     </li>
 </ul>
-<h4>9. Data protection provisions about the application and use of Facebook</h4>
-<p>On this website, the controller has integrated components of the enterprise
+<!-- This needs to be revisited as the only interaction peaople have with facebook and/or twitter
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
<!-- This needs to be revisited as the only interaction people have with facebook and/or twitter 
```

File: web/privacy.html [hunk 9]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
+        <p>In addition, you have the right, on grounds relating to your particular situation, to object to processing of personal data concerning yourself by joind.in for scientific or historical research purposes, or for statistical purposes pursuant to Article 89(1) of the GDPR, unless the processing is necessary for the performance of a task carried out for reasons of public interest.</p>
 
+        <p>In order to exercise the right to object, you may contact the joind.in-team via the contact form. In addition, you are free in the context of the use of information society services, and notwithstanding Directive 2002/58/EC, to use your right to object by automated means using technical specifications.</p>
     </li>
+   
     <li><h4>i) Right to withdraw data protection consent </h4>
-        <p>Each data subject shall have the right granted by the European legislator to withdraw his or her consent to processing of his or her personal data at any time. </p>
+        <p>you shall have the right granted by the European legislator to withdraw your consent to processing of his or her personal data at any time. </p>
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
        <p>You shall have the right granted by the European legislator to withdraw your consent to processing of your personal data at any time. </p>
```

File: web/privacy.html [hunk 10]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
 
-        <p>Furthermore, in exercising his or her right to data portability pursuant to Article 20(1) of the GDPR, the data subject shall have the right to have personal data transmitted directly from one controller to another, where technically feasible and when doing so does not adversely affect the rights and freedoms of others.</p>
+        <p>Furthermore, in exercising your right to data portability pursuant to Article 20(1) of the GDPR, you shall have the right to have personal data transmitted directly from one controller to another, where technically feasible and when doing so does not adversely affect the rights and freedoms of others.</p>
 
-        <p>In order to assert the right to data portability, the data subject may at any time contact any employee of the joind.in.</p>
+        <p>In order to assert the right to data portability, you may at any time contact the joind.in-Team via the contact form.</p>
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
        <p>In order to assert the right to data portability, you may at any time contact the joind.in-team via the contact form.</p>
```

File: web/privacy.html [hunk 11]
```diff
@@ -249,95 +237,89 @@ <h4>8. Rights of the data subject</h4>
...
 
+        <p>In order to exercise the right to object, you may contact the joind.in-team via the contact form. In addition, you are free in the context of the use of information society services, and notwithstanding Directive 2002/58/EC, to use your right to object by automated means using technical specifications.</p>
     </li>
+   
     <li><h4>i) Right to withdraw data protection consent </h4>
-        <p>Each data subject shall have the right granted by the European legislator to withdraw his or her consent to processing of his or her personal data at any time. </p>
+        <p>you shall have the right granted by the European legislator to withdraw your consent to processing of his or her personal data at any time. </p>
 
-        <p>f the data subject wishes to exercise the right to withdraw the consent, he or she may, at any time, contact any employee of the joind.in.</p>
+        <p>If you wish to exercise the right to withdraw the consent, you may, at any time, contact the joind.in-Team via the contact form.</p>
```
Comment (ericpoe) [inline web/privacy.html]:
```suggestion
        <p>If you wish to exercise the right to withdraw the consent, you may, at any time, contact the joind.in-team via the contact form.</p>
```

**Comment by heiglandreas** [comment]
Obsoloeted by #802

**Pull request closed without merging**